src/main/resources/application.properties

```diff
@@ -36,7 +36,3 @@ helium.keyvault.secretName =MySecret
 management.endpoints.web.base-path=/
 management.endpoints.web.path-mapping.health=healthz
 
-azure.cosmosdb.key=
-azure.cosmosdb.database=
-azure.cosmosdb.uri=
-
```

Comment:
Also, can you remove the ActorsRepository.java file from the project. It should not be needed anymore, right?

Reply:
Removed all of these, they are not needed anymore

Reply:
Yay!